package.json

```diff
@@ -4,24 +4,81 @@
   "private": true,
   "description": "Website for KYBING PVT LTD",
   "homepage": "https://kybing.github.io",
+  "scripts": {
+    "start": "node scripts/start.js",
+    "build": "node scripts/build.js",
+    "test": "node scripts/test.js",
+    "deploy": "gh-pages -d build",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix ."
+  },
   "dependencies": {
     "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.3.2",
-    "@testing-library/user-event": "^7.1.2",
+    "babel-loader": "8.1.0",
+    "babel-plugin-named-asset-import": "^0.3.6",
+    "camelcase": "^5.3.1",
```

Comment:
I think this can go to dev dependencies

Reply:
I checked this it will not go into devDependency, check out this link for more info - https://www.npmjs.com/package/camelcase